src/main/java/com/google/devtools/build/lib/runtime/BlazeCommandDispatcher.java

```diff
@@ -619,7 +625,10 @@ private BlazeCommandResult execExclusively(
       }
 
       // Parse starlark options.
-      earlyExitCode = optionHandler.parseStarlarkOptions(env, reporter);
+      try (SilentCloseable c =
+          Profiler.instance().profile(ProfilerTask.BZLMOD, "parse starlark options")) {
+        earlyExitCode = optionHandler.parseStarlarkOptions(env);
```

Comment:
```suggestion
        earlyExitCode = optionHandler.parseStarlarkOptions(env, reporter);
```

CC @iancha1992

Reply:
@fmeum Thanks! I was just going to close this because I thought we'd need another commit to cherry-pick it. But if this fixes it, then I guess it should be good!